app/models/bgs_dependents/child.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module BGSDependents
+  class Child < Base
```

Comment:
Please document class attributes and methods with [YARD](https://yardoc.org/)

Reply:
Added comments for YARD documentation.